deepchecks/tabular/checks/train_test_validation/train_test_feature_drift.py

```diff
@@ -81,11 +81,15 @@ class TrainTestFeatureDrift(TrainTestCheck, ReduceMixin):
     ignore_na: bool, default True
         For categorical columns only. If True, ignores nones for categorical drift. If False, considers none as a
         separate category. For numerical columns we always ignore nones.
-    aggregation_method: str, default: "weighted"
+    aggregation_method: str, default: "l2_combination"
         argument for the reduce_output functionality, decides how to aggregate the drift scores for a
-        collective score. Possible values are:
+        collective score. The collective score value is between 0 and 1 for all methods other than l2_combination.
+        Possible values are:
         'weighted': Weighted mean based on feature importance, provides a robust estimation on how
         much the drift will affect the model's performance.
+        'l2_combination': L2 norm over the combination of drift scores and feature importance, minus the
```

Comment:
Maybe workshop the name a bit? I don't feel like I would get this out of this name.